src/components/types.ts

```diff
@@ -32,7 +32,8 @@ export interface InputProps extends InputStatus, InputHTMLAttributes<HTMLInputEl
    */
   endIcon?: ReactNode;
 
-  weight?: number;
+  fontWeight?: number;
```

Comment:
Shouldn't this be a string given how it's used here:

```
${({ theme, fontWeight = 'regular' }) => theme.typography.weights[fontWeight]};
```

Reply:
no, fontWeight is a number https://github.com/zopaUK/react-components/blob/master/src/components/styles/Theme/ThemeProvider.tsx#L241

Reply:
https://github.com/zopaUK/react-components/blob/master/src/constants/typography.ts#L32

Reply:
InputProps is being used as the interface for this:

https://github.com/zopaUK/react-components/pull/860/files#diff-7fba51a07023554f00291cb6d526da934112691ee58c2ed0e1f37ffe3e8bd60eR50

and the default here: `fontWeight = 'regular'` is a string which makes no sense

Reply:
i've made changes to this now :)